### DISCUSSION TITLE
Requiring Chat Extension Is Lame

### DISCUSSION THREAD

**Discussion by mjmatthiesen** [community/community#69845] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Previously I had only the copilot extension installed, now with a recent update, it has forced me to install the chat application. I understand that some people may find this useful, but the chat is slow and cumbersome, I rarely if ever user that. If I uninstall the extension, copilot is also removed. Same goes for disable. It appears a new dependency, opposite of what is sensible, was introduced.

I didn't ask for this "feature" so I am hoping you revert this so I can continue on without an unnecessary extension.

**Comment by Mini-ghost** [comment]
I have the same problem. I need Intellisense when I press `Command` + `I`, or there should be some other way to open the chat.
